### ISSUE TITLE
[Request] Disabling items / recipes

### ISSUE THREAD

**Issue by TDarkShadow** [Brian-Wuest/MC-Prefab#34]
Would it be possible to have a config where you can disable items / recipes of the mod?
Atm, I use CraftTweaker scripts to disable / alter many recipes and even some items to have a reason to explore villages / nether fortresses and give the players a rewarding loot.
However, CraftTweaker JEI integration is broken and I also would like that the players can't see certain items in Creative Mode.

**Comment by Brian-Wuest** [comment]
I can work on adding a configuration to disable/enable recipes. I never added that functionality because no one asked for it 😃.

**Comment by TDarkShadow** [comment]
Thank you very much!
If it is possible, also to disable some items entirely?

**Comment by Brian-Wuest** [comment]
I can certainly disable the recipe, however I don't think I can truely disable and re-enable an item or block without causing a complete restart of minecraft. I have done this in WuestUtilities before with the blocks and items provided in that mod.

If you are ok with a restart of minecraft then I can do that.

See this post on the Minecraft Forum for official answer to unregistering an item. http://www.minecraftforge.net/forum/topic/26269-unregistering-items-from-itemregistry/

**Comment by Brian-Wuest** [comment]
So, I was able to get the enable/disable recipe processing to work while still in the game. However, recipe listing mods just as Not Enough Items (NEI), Just Enough Items (JEI) will not update the recipe list automatically without adding a dependency on the JEI API which I don't want to do.

I will have to require a restart of minecraft when changing these settings.

**Also:** Because of how integrated some of these recipes are...I don't want to completely not-register blocks and items as some blocks/items would not be able to be crafted at all if the user forgot the disable later items/blocks.

**For Example:** The server admin has removed the Chicken Coop from the registry but not the Advanced Chicken Coop (probably due to user error but still). This would cause an error during recipe registration.

I have grouped together certain recipes in the enable/disable configuration. These are typically the compressed items.
**Example:** Compressed Stone, Double Compressed Stone, Triple Compressed Stone

**TLDR;**
Each block/item recipe can be disabled. Requires Minecraft restart.

**Comment by TDarkShadow** [comment]
Thank you very much.
I don't mind to restart Minecraft, it is mostly for a modpack I'm making with my friends anyway.

**Comment by Brian-Wuest** [comment]
This has been implemented in version 1.1.1.11 for Minecraft 1.10.2 and 1.2.0.6 for Minecraft 1.11.2.

These are available on curseforge.
